### PR TITLE
[WIP] Update core

### PR DESCRIPTION
**Pull request by daniel-beck**

### CODE REVIEW

**Comment by daniel-beck** [comment]
@batmat Could you look into this? I expect that https://github.com/jenkinsci/jenkins/pull/2998 broke `site:site`.

File: Jenkinsfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ node('docker') {
         /* Checkout the latest LTS release so we don't need to build Jenkins to create the site */
         sh 'git clone https://github.com/jenkinsci/jenkins/ jenkins'
         dir ('jenkins') {
-            sh 'git checkout jenkins-2.46'
+            sh 'git checkout jenkins-2.78'
```
Comment (batmat) [inline Jenkinsfile]: I'm discovering this repo TBH. Care to fix/clarify the comment above this? 2.46 was an LTS, 2.78 is not, so merging as-is would make the comment go moot and very misleading to readers.

Reading the commit logs didn't help either. What is the intent here? How does it fail? I think we ought to store more things here about the failure for reference.

**Comment by batmat** [comment]
I _think_ I might understand here what you're doing. Your intent is not to merge this right? Just you manually did some sort of _bisect_ to identify the cause of the failure, and now you'd like me to try and fix it durably?

Thanks

**Comment by batmat** [comment]
Note: for whatever reason, I'm unable to successfully run `mvn -DgenerateProjectInfo=false -DgenerateSitemap=false -e clean site:site` on my laptop on jenkins 2.46.

**Comment by daniel-beck** [comment]
@batmat 

> Your intent is not to merge this right? Just you manually did some sort of bisect to identify the cause of the failure, and now you'd like me to try and fix it durably? If so, then can you please file a JIRA and assign it to me?

Yes, since #4 failed.

Filing core issue.

**Comment by batmat** [comment]
See https://issues.jenkins-ci.org/browse/JENKINS-50969 for tracking

**Comment by batmat** [comment]
Fixed in https://github.com/jenkins-infra/core-taglibs-report-generator/pull/7

**Pull request closed without merging**